utils/setStatus.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const statusCodes = require('http').STATUS_CODES
+    , isUndefined = require('./tools').isUndefined
+    , setStatus = () => {
+        return function (status) {
```

Comment:
Since it isn't needed it would just tie up a little more memory for every request.